packages/yourloops/test/integration/assert/monitoring-alerts-parameters.assert.ts

```diff
@@ -59,6 +60,57 @@ export const checkMonitoringAlertsDialogContentMmol = async (): Promise<void> =>
   expect(screen.queryByRole('dialog')).not.toBeInTheDocument()
 }
 
+export const checkMonitoringAlertsParametersTeamAdmin = async (): Promise<void> => {
+  jest.spyOn(TeamApi, 'editTeam').mockResolvedValue(undefined)
+  const monitoringAlertsParameters = await screen.findByTestId('team-monitoring-alerts-configuration')
+  expect(monitoringAlertsParameters).toHaveTextContent('Monitoring alerts configuration1. Time away from target rangeCurrent trigger setting: 5% of time off target (min at 50 mg/dL max at 140 mg/dL)A. Glycemic targetMinimum:​mg/dLMaximum:​mg/dLDefault: min at 70 mg/dL and max at 180 mg/dLB. Event trigger thresholdTime spent off target5%​Default: 50%')
+  expect(monitoringAlertsParameters).toHaveTextContent('2. Severe hypoglycemiaCurrent trigger setting: 10% of time below 40 mg/dL thresholdA. Severe hypoglycemia threshold:Severe hypoglycemia below:​mg/dLDefault: 54 mg/dLB. Event trigger thresholdTime spent in severe hypoglycemia10%​Default: 5%')
+  expect(monitoringAlertsParameters).toHaveTextContent('3. Data not transmittedCurrent trigger setting: 15% of data not transmitted over the periodA. Event trigger thresholdTime spent without uploaded data15%​Default: 50%Save')
+
+  const lowBgInput = within(monitoringAlertsParameters).getByRole('spinbutton', { name: 'Low blood glucose input' })
+  const highBgInput = within(monitoringAlertsParameters).getByRole('spinbutton', { name: 'High blood glucose input' })
+  const veryLowBgInput = within(monitoringAlertsParameters).getByRole('spinbutton', { name: 'Very low blood glucose input' })
+
+  const outOfRangeThreshold = within(monitoringAlertsParameters).getByTestId('basic-dropdown-out-of-range-selector')
+  const hypoThreshold = within(monitoringAlertsParameters).getByTestId('basic-dropdown-hypo-threshold-selector')
+  const nonDataTxThreshold = within(monitoringAlertsParameters).getByTestId('basic-dropdown-non-data-selector')
+
+  expect(lowBgInput).not.toBeDisabled()
+  expect(highBgInput).not.toBeDisabled()
+  expect(veryLowBgInput).not.toBeDisabled()
+  expect(outOfRangeThreshold).not.toHaveClass('Mui-disabled')
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏It could be nice to create a const for this value, given that it's used multiple times in the 2 functions